modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -31,19 +31,21 @@ $ oc edit ingresses.config/cluster -o yaml
 +
 .. Edit the YAML file:
 +
-.Sample `appsDomain` configuration to `apps.acme.io`
+.Sample `appsDomain` configuration to `hello_openshift_my_project.apps.acme.io`
 [source,yaml]
 ----
 apiVersion: config.openshift.io/v1
 kind: Ingress
 metadata:
   name: cluster
 spec:
-  domain: apps.<domain_url>
-  appsDomain: apps.acme.io
+  domain: apps.default_domain.openshift.com                <1>
+  appsDomain: <hello_openshift_my_project.apps.acme.io>    <2>
```

Comment:
`<hello_openshift_my_project>.apps.acme.io`  looks like a route host name instead of domain name.`
`domain` field specify the default domain that like `apps.example.com`
`appsDomain` field is optional that allow user to use other domain prefix other than `apps`,  e.g `test.example.com`.
After applying the change, then new create route will use the hostname like `hello-openshift-my_project.test.example.com`.

Reply:
we can use below setting for the sample.
```
spec:
  domain: apps.example.com
  appsDomain: test.example.com
```